chainerx_cc/chainerx/strides.cc

```diff
@@ -66,4 +66,19 @@ std::tuple<int64_t, int64_t> GetDataRange(const Shape& shape, const Strides& str
     return std::tuple<int64_t, int64_t>{first, last};
 }
 
+int64_t GetDataSize(const Shape& shape, const Strides& strides, size_t item_size) {
```

Comment:
Do we need this function?
It's quite straight-forward to use `GetDataRange` IMO (and it's not used so often).